service/raft/raft_group0.cc

```diff
@@ -592,6 +592,10 @@ future<> raft_group0::setup_group0(
     // Perform a Raft read barrier so we know the set of group 0 members and our group 0 state is up-to-date.
     co_await group0_server.read_barrier(&_abort_source);
 
+    utils::get_local_injector().inject("stop_after_joining_group0", [&] {
+        _abort_source.request_abort();
```

Comment:
This won't cause the execution to stop here, it will proceed until the next usage of `_abort_source`.
Better to `throw runtime_error` or something.

Reply:
Also, why do it here, and not immediately after `co_await join_group0(...)`?

Reply:
> Also, why do it here, and not immediately after co_await join_group0(...)?

Without specific reason, I made it here to match logs from #13543.

Reply:
Done. (both comments)